src/Raven.Studio/typescript/models/database/tasks/ongoingTaskEmbeddingsGenerationEditModel.ts

```diff
@@ -328,12 +351,13 @@ class ongoingTaskEmbeddingsGenerationEditModel extends ongoingTaskEditModel {
             Transforms: null,
             Collection: this.collectionInput(),
             ChunkingOptionsForQuerying: {
+                OverlapTokens: this.overlapTokens() ?? 0,
```

Comment:
You used the same `overlapTokens` in 3 places. Those should be 3 different fields